guides/rails_guides/markdown/renderer.rb

```diff
@@ -68,14 +68,14 @@ def convert_footnotes(text)
 
         def lexer_language(code_type)
           case code_type
-          when "css", "js", "html", "ruby", "sql", "yaml"
-            code_type
-          when "erb", "html+erb"
+          when "html+erb"
             "erb"
           when "bash"
             "console"
-          else
+          when nil
             "plaintext"
+          else
+            code_type
```

Comment:
If this is a syntax Rouge doesn't understand, `lexer` will be nil on line 15, which we'll need to handle properly.

Reply:
That's a very good point @eugeneius. I've added a commit to address this: https://github.com/rails/rails/pull/39901/commits/482834dce0354308c8ceb9c3a71006ba769b3d9d